js/fuzzyset.js

```diff
@@ -0,0 +1,281 @@
+var FuzzySet = (function () {
```

Comment:
Wasn't sure how best to include this library, so I copied and pasted it for now.